src/agent/DiskId.hpp

```diff
@@ -0,0 +1,15 @@
+
+#pragma once
+
+#include <string>
+
+class DiskId
+{
+public:
+    DiskId(const std::string& systemId);
+
+    std::string getSystemId();              ///< returns system specific disk identifier (/dev/sdX on linux and X: on Windows)
```

Comment:
Collecting system disc identifier should be adressed as Enhancement issue. I think that since now it's good to create such issues.